scripts/bundle-build/standard-gradle-build/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+outputDir=$(basename $PWD)-artifacts
+mkdir $outputDir
+./gradlew assemble --no-daemon --refresh-dependencies -Dbuild.snapshot=false -DskipTests=true -Dopensearch.version=$1
+
+zipPath=$(find . -path \*build/distributions/*.zip)
```

Comment:
Error handling if there are no zips?

Reply:
Maybe I should have named this file path differently,  This script will only be invoked for components that do not have a special case and product a zip at a build/distributions location.  If the event something is updated and there is no zip, then it should fail.

Reply:
I'll replace this to be picking up files explicitly.